docs/native-code.md

```diff
@@ -57,4 +57,4 @@ Any libraries you use should be built as WinRT components. In other words, you c
   - Libraries built for .NET desktop framework cannot be directly accessed by UWP. You can create a .NET Standard library that calls into the .NET framework one, and call from the UWP app into the .NET Standard middleware.
 
 ### Local testing and inner loop
-For internal development, you can deploy your app for test purposes by side-loading and deploying via [loose-file registration](https://docs.microsoft.com/windows/uwp/debug-test-perf/loose-file-registration). When building in **Debug** mode (which is the default), `npx react-native run-windows` performs loose-file registration of your app in order to install it locally. When running `npx react-native run-windows` with the `--release` switch, the CLI will install the real package onto your local machine. This requires the app to be signed and for the certificate it uses for signing to be trusted by the machine the app is going to be installed on. See [Create a certificate for package signing](https://docs.microsoft.com/windows/msix/package/create-certificate-package-signing) and [Intro to certificates](https://docs.microsoft.com/windows/uwp/security/certificates).
+For internal development, you can deploy your app for test purposes by side-loading and deploying via [loose-file registration](https://docs.microsoft.com/windows/uwp/debug-test-perf/loose-file-registration). When building in **Debug** mode (which is the default), `npx react-native run-windows` performs loose-file registration of your app in order to install it locally. When running `npx react-native run-windows` with the `--release` switch, the CLI will install the real package onto your local machine. This requires the app to be signed and for the certificate it uses for signing to be trusted by the machine the app is going to be installed on. See [Create a certificate for package signing](https://docs.microsoft.com/windows/msix/package/create-certificate-package-signing) and [Intro to certificates](https://docs.microsoft.com/windows/uwp/security/certificates). You may also choose to run `npx react-native run-windows --deploy-from-layout`. Here, the CLI will install the package onto your local machine without the need for a certificate.

```

Comment:
note that Jon has a PR out where we might make the default for release be layout deployment too:  #556

Reply:
Cool will keep an eye on the PR and can re-update the docs when that change is merged!

Reply:
right, my point is that I think we should change the default in the CLI to be deploy from layout, since otherwise it's easy to run into a sharp corner. 
That means 
a) changing the CLI to always use deployrecipe, 
b) changing the PR to not pass --deploy-from-layout, 
c) create a new option to opt out of deploy from layout and create an app package one can submit to the store 
d) update the doc here

Reply:
Here's a PR to change the CLI: https://github.com/microsoft/react-native-windows/pull/8776

Reply:
From that PR, the CLI will now *always* deploy from layout if:

- passed `--deploy-from-layout` (to force it) *OR*
- the build had `AppxSigningEnabled` missing or set to false *OR*
- the build had `PackageCertificateFile` not set

Alternatively, you can say we only try to deploy via package if *BOTH* `AppxSigningEnabled` is set to true *AND* `PackageCertificateFile` is set.

Reply:
Great will close out this PR now, as changes are no longer needed!